src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator-with-skip.test.tsx.snap

```diff
@@ -490,7 +490,7 @@ exports[`authenticator-enroll-select-authenticator renders form 1`] = `
                       tabindex="0"
                       type="submit"
                     >
-                      Set up later
+                      Continue
```

Comment:
The i18n key was updated months ago, not sure why this test was never updated, nor do I understand how it ever passed 😅